src/js/lib/create-classname-string.js

```diff
@@ -0,0 +1,9 @@
+export default (...classes) =>
```

Comment:
Wouldn't it be better to use a named function as you're exporting it? Makes debugging on errors a little easier.